quackfactory/cogs/session.py

```diff
@@ -0,0 +1,34 @@
+from discord.ext import commands
+import discord
+import logging
+
+intents = discord.Intents.default()
+intents.reactions = True
+intents.members = True
+client = discord.Client(intents=intents)

```

Comment:
Why are you creating a separate instance of a bot inside of the cog? I recommend to go through the project and follow the code flow. Declaring intents and creating a new bot instance isn't what cogs are for -- these things are already created inside of the application.

PS: You are importing the logging package, yet you aren't using it.

Reply:
i'll remove those 👍